src/python_testing/TC_DRLK_2_13.py

```diff
@@ -0,0 +1,656 @@
+                                    "Unexpected return type for SetCredential")
+                asserts.assert_true(response.status == Status.Success,
+                                    "Error sending SetCredential command, status={}".format(str(response.status)))
+            except InteractionModelError as e:
+                logging.exception(e)
+                asserts.assert_equal(e.status, Status.Success, f"Unexpected error returned: {e}")
+
+    async def read_and_validate_lock_event_change(self, verify_lock_data_type, priority, operation_source,
+                                                  data_type_operation, fabric_index, node_id, data_index, user_index):
+        response_event_list = await self.default_controller.ReadEvent(self.dut_node_id,
+                                                                      [1, cluster.Events.LockUserChange])
+        for event_read_result in reversed(response_event_list):
+            if not hasattr(event_read_result.Data, "lockDataType"):
+                continue
+            if event_read_result.Data.lockDataType == verify_lock_data_type:
+                asserts.assert_equal(event_read_result.Header.Priority, priority,
+                                     f"Priority mismatch, Priority={event_read_result.Header.Priority}")
+                asserts.assert_equal(event_read_result.Data.operationSource, operation_source,
+                                     f"OperationSource not matching, operationSource={operation_source}")
+                asserts.assert_equal(event_read_result.Data.fabricIndex, fabric_index,
+                                     f"FabricIndex not matching, fabricIndex={fabric_index}")
+                asserts.assert_equal(event_read_result.Data.dataIndex, data_index,
+                                     f"DataIndex not matching, dataIndex={data_index}")
+                asserts.assert_equal(event_read_result.Data.sourceNode, node_id,
+                                     f"SourceNode not matching, sourceNode={node_id}")
+                asserts.assert_equal(event_read_result.Data.userIndex, user_index,
+                                     f"UserIndex not matching, userIndex={user_index}")
+                asserts.assert_equal(event_read_result.Data.dataOperationType, data_type_operation,
+                                     f"DataOperationType not matching, dataOperationType={data_type_operation}")
+                break
+
+    async def clear_credentials_cmd(self, credential, step=None, expected_status: Status = Status.Success):
+        try:
+            if step is not None:
+                self.step(step)
+            if self.pics_guard(self.check_pics("DRLK.S.F0d") and self.check_pics("DRLK.S.C26.Rsp")):
+                await self.send_single_cmd(cmd=Clusters.DoorLock.Commands.ClearCredential(credential=credential),
+                                           endpoint=1,
```

Comment:
could we get a constant for endpoint id that is used everywhere? Or is endpointid 1 mandatory to be the only endpoint supported for this test (by spec)?

Reply:
started using ` self.app_cluster_endpoint = 1`  as constant value in all places